Found it. It appears that there are meter references that need to be updated in `p4_info_manager_mock.h`.
```text
                 from stratum/hal/lib/p4/p4_action_mapper_test.cc:14:
./stratum/hal/lib/p4/p4_info_manager_mock.h:73:47: error: ISO C++ forbids declaration of 'type name' wit
h no type [-fpermissive]
   73 |                      ::util::StatusOr<const ::p4::config::v1::PacketModMeter>(
      |                                               ^~
./stratum/hal/lib/p4/p4_info_manager_mock.h:73:77: error: template argument 1 is invalid
   73 |                      ::util::StatusOr<const ::p4::config::v1::PacketModMeter>(
      |                                                                             ^
./stratum/hal/lib/p4/p4_info_manager_mock.h:72:3: error: template argument 1 is invalid
   72 |   MOCK_CONST_METHOD1(FindPktModMeterByID,
      |   ^~~~~~~~~~~~~~~~~~
./stratum/hal/lib/p4/p4_info_manager_mock.h:72:3: error: template argument 1 is invalid
   72 |   MOCK_CONST_METHOD1(FindPktModMeterByID,
      |   ^~~~~~~~~~~~~~~~~~
./stratum/hal/lib/p4/p4_info_manager_mock.h:73:47: error: ISO C++ forbids declaration of 'type name' wit
h no type [-fpermissive]
   73 |                      ::util::StatusOr<const ::p4::config::v1::PacketModMeter>(
      |                                               ^~
./stratum/hal/lib/p4/p4_info_manager_mock.h:73:77: error: template argument 1 is invalid
   73 |                      ::util::StatusOr<const ::p4::config::v1::PacketModMeter>(
      |                                                                             ^
./stratum/hal/lib/p4/p4_info_manager_mock.h:72:3: error: template argument 1 is invalid
   72 |   MOCK_CONST_METHOD1(FindPktModMeterByID,
      |   ^~~~~~~~~~~~~~~~~~
./stratum/hal/lib/p4/p4_info_manager_mock.h:72:3: error: template argument 1 is invalid
   72 |   MOCK_CONST_METHOD1(FindPktModMeterByID,
      |   ^~~~~~~~~~~~~~~~~~
./stratum/hal/lib/p4/p4_info_manager_mock.h:72:3: error: template argument 1 is invalid
   72 |   MOCK_CONST_METHOD1(FindPktModMeterByID,
      |   ^~~~~~~~~~~~~~~~~~
In file included from external/com_google_googletest/googlemock/include/gmock/gmock-actions.h:147,
                 from external/com_google_googletest/googlemock/include/gmock/gmock.h:59,
                 from stratum/hal/lib/p4/p4_action_mapper_test.cc:14:
./stratum/hal/lib/p4/p4_info_manager_mock.h:73:47: error: ISO C++ forbids declaration of 'type name' wit
h no type [-fpermissive]
```

The actual error log is huge. I suspect this is why Bazel didn't display it.

-----

Two tech tips:

1. If you're changing a symbol, it's a good idea to grep the codebase for occurrences you've missed.
2. Any time you work on a class in Stratum, there are generally four associated files:
   - Source file (`wombat.cc`)
   - Header file (`wombat.h`)
   - Unit test (`wombat_test.cc`)
   - Mock (`wombat_mock.h`)

--- a/stratum/hal/lib/p4/BUILD
+++ b/stratum/hal/lib/p4/BUILD
@@ -79,7 +79,6 @@ stratum_cc_test(
         "//stratum/public/proto:p4_annotation_cc_proto",
         "//stratum/public/proto:p4_table_defs_cc_proto",
         "@com_github_p4lang_p4runtime//:p4info_cc_proto",
-        "@com_github_p4lang_p4runtime//:idpf_p4info_cc_proto",
         "@com_google_absl//absl/memory",
         "@com_google_absl//absl/strings",
         "@com_google_googletest//:gtest_main",

--- a/stratum/hal/lib/p4/p4_info_manager_mock.h
+++ b/stratum/hal/lib/p4/p4_info_manager_mock.h
@@ -70,20 +70,20 @@ class P4InfoManagerMock : public P4InfoManager {
 
   // FindPktModMeter
   MOCK_CONST_METHOD1(FindPktModMeterByID,
-                     ::util::StatusOr<const ::p4::config::v1::PacketModMeter>(
+                     ::util::StatusOr<const ::idpf::PacketModMeter>(
                          uint32 meter_id));
   MOCK_CONST_METHOD1(FindPktModMeterByName,
-                     ::util::StatusOr<const ::p4::config::v1::PacketModMeter>(
+                     ::util::StatusOr<const ::idpf::PacketModMeter>(
                          const std::string& meter_name));
 
   // FindDirectPktModMeter
   MOCK_CONST_METHOD1(
       FindDirectPktModMeterByID,
-      ::util::StatusOr<const ::p4::config::v1::DirectPacketModMeter>(
+      ::util::StatusOr<const ::idpf::DirectPacketModMeter>(
           uint32 meter_id));
   MOCK_CONST_METHOD1(
       FindDirectPktModMeterByName,
-      ::util::StatusOr<const ::p4::config::v1::DirectPacketModMeter>(
+      ::util::StatusOr<const ::idpf::DirectPacketModMeter>(
           const std::string& meter_name));
 
   // FindValueSet

--- a/stratum/hal/lib/p4/p4_info_manager_mock.h
+++ b/stratum/hal/lib/p4/p4_info_manager_mock.h
@@ -69,9 +69,9 @@ class P4InfoManagerMock : public P4InfoManager {
                          const std::string& meter_name));
 
   // FindPktModMeter
-  MOCK_CONST_METHOD1(FindPktModMeterByID,
-                     ::util::StatusOr<const ::idpf::PacketModMeter>(
-                         uint32 meter_id));
+  MOCK_CONST_METHOD1(
+      FindPktModMeterByID,
+      ::util::StatusOr<const ::idpf::PacketModMeter>(uint32 meter_id));
   MOCK_CONST_METHOD1(FindPktModMeterByName,
                      ::util::StatusOr<const ::idpf::PacketModMeter>(
                          const std::string& meter_name));
@@ -79,12 +79,10 @@ class P4InfoManagerMock : public P4InfoManager {
   // FindDirectPktModMeter
   MOCK_CONST_METHOD1(
       FindDirectPktModMeterByID,
-      ::util::StatusOr<const ::idpf::DirectPacketModMeter>(
-          uint32 meter_id));
-  MOCK_CONST_METHOD1(
-      FindDirectPktModMeterByName,
-      ::util::StatusOr<const ::idpf::DirectPacketModMeter>(
-          const std::string& meter_name));
+      ::util::StatusOr<const ::idpf::DirectPacketModMeter>(uint32 meter_id));
+  MOCK_CONST_METHOD1(FindDirectPktModMeterByName,
+                     ::util::StatusOr<const ::idpf::DirectPacketModMeter>(
+                         const std::string& meter_name));
 
   // FindValueSet
   MOCK_CONST_METHOD1(

